test/Olcs/src/FormService/Form/Lva/ApplicationFinancialEvidenceTest.php

```diff
@@ -37,14 +43,102 @@ class ApplicationFinancialEvidenceTest extends MockeryTestCase
     /** @var  m\MockInterface */
     protected $translator;
 
+    /**
+     * We can access service manager if we need to add a mock for certain applications
+     *
+     * @return \Laminas\ServiceManager\ServiceLocatorInterface
+     */
+    public function getServiceManager()
```

Comment:
Just curious on the thinking here? Presuming it's to do with removing Bootstrap.php and the reliance on olcs-teswthelpers repo, but getServiceManager and getRealServiceManager are now duplicated in a lot of places...

Reply:
I asked for this; It's the same for all migrated repositories (or will be). Removing bootstrap files and the complexity that comes with them seemed the lesser evil at the cost of duplication.

Reply:
A lot of duplication though - common will have a number of them and so will internal. Especially with the FormElementManager stuff, we're seeing a lot of failing form tests on project/3.0 branches due to this code, so it does leave us perhaps needing to update in a lot of places in the near future to get things running.

This is code which is already problematic as it is, so maybe we find a way to engineer it out instead of duplicating?

Reply:
Can be revisited in a future ticket if you have a better solution? The removal of the Bootstrap file was the AC for this ticket.

Longer term the source/test code would get refactored to not use the service manager and mock/DI instead, unit tests shouldn't be touching real vendor code. The majority of the factory unit tests are also going to be deleted in the near future which would remove a lot of these service-manager-type method calls.